src/Core/WheelMUD.Core.csproj

```diff
@@ -54,6 +57,7 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
I'm not up to date on what has been going on in .NET with architectures for the last few years...  I tried to read up a bit now but it's late and my head hurts.  At a glance though this feels like it should be true?

(Ultimately in the long run I think I'd prefer a single Any CPU option, where an x64 system would be allowed to run in 64 bit mode, instead of x86/x64 options... but that has implications on how we handle dependencies and is not always trivial to accomplish well.  May be worth an issue to track exploring this?)

Reply:
It appears to be a setting to have the IL compile to x86 on a 64-bit machine. Since we have all the projects set to compile to x86 the setting is not applicable so its false in all the .proj files. If we change the projects to AnyCPU then it might matter if we use an unmanaged assembly that only supports x86 architecture.

There's more information [here](https://dzone.com/articles/what-anycpu-really-means-net).

Reply:
(Seems weird they default it false here instead of omitting.  Oh well.)

Anyway, if we're abandoning RavenDB instead, I wonder if we still want the project targeting upped or not.  I checked that we still open and run fine in VS2015.  I don't have anything earlier on my PC, but I'd be fine even if VS2015 was the earliest we were to support anymore.  I'm thinking to take this merge.